src/server/auth/server/testing/auth_test.go

```diff
@@ -2481,7 +2481,6 @@ func TestRestoreAuthToken(t *testing.T) {
 	require.True(t, auth.IsErrExpiredToken(err))
 
 	// restore a token with an expiration date in the future
-	req.Token.HashedToken = fmt.Sprintf("%x", sha256.Sum256([]byte("expiring-token")))
```

Comment:
Why is this being removed?

Reply:
I'm going to remove this change from the PR - i previously had other changes in this file too. 

I originally removed it because technically this line isn't doing anything since this value is being set a few lines earlier